### PR TITLE
Revise Reporting Bad Behaviour folder

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/pt-br.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
+# Handling Foul Play (Español)
```
Comment (Joehuu) [inline wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/pt-br.md]: This is not Espanol.

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/pt-br.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
+# Handling Foul Play (Português (Brasil))
```
[the remark this answers is not in the document]
Comment (TPGPL) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/pt-br.md]: yeah `# Original Title (Language)`

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md [hunk 3]
```diff
@@ -1,32 +1,48 @@
-==Pourquoi suis-je banni ?==
+# Triche et bannissement
```
[the remark this answers is not in the document]
Comment (TPGPL) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md]: it was like this on the old wiki so i would leave it as it is

[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md]: k

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md [hunk 4]
```diff
@@ -36,20 +36,18 @@ Restez calme. **N'en parlez pas dans le chat public.** Ne donnez ces information
...
 
 ## Comment puis-je demander mon débannissement en cas d'erreur ?
 
-Envoyez un mail à <accounts@ppy.sh> et utilisez le sujet "Ban Appeal: <username>".
-Ajoutez la raison de votre bannissement, et expliquez pourquoi votre comportement ou manière de jouer n'aurait pas dû conduire à un bannissement.
-Attendez une semaine maximum pour une réponse. Si votre demande de débanissement a été rejetée, toutes vos prochaines demandes seront rejetées, donc appliquez-vous dans la première.
+Envoyez un mail à <accounts@ppy.sh> et utilisez le sujet `Ban Appeal: <username>`.
+Ajoutez la raison de votre bannissement, et expliquez pourquoi votre comportement ou manière de jouer n'aurait pas dû conduire à un bannissement. Attendez une semaine maximum pour une réponse. Si votre demande de débanissement a été rejetée, toutes vos prochaines demandes seront rejetées, donc appliquez-vous dans la première.
```
Comment (cl8n) [inline wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md]: still has the same issue

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/zh.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ osu!使用与人工确认相结合的自动作弊探测。
 不过，除非站在你旁边，不然我们显然很难监视你，但如果你被指控有这种行为，录制带有手元的打图过程可以帮你证明清白。
 
 ## 如何申诉解除限制？
-在最后一次限制后等三个月再提请申诉。在申请用"Ban Appeal: <username>"（封禁申诉：用户名）为标题给 [accounts@ppy.sh](mailto:accounts@ppy.sh) 发邮件。
+在最后一次限制后等三个月再提请申诉。在申请用`Ban Appeal: <username>`（封禁申诉：用户名）为标题给 [accounts@ppy.sh](mailto:accounts@ppy.sh) 发邮件。
 团队希望用户提供他们行为的所有历史并且给出他们不会再次违规的原因。
```
Comment (cl8n) [inline wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/zh.md]: same thing

**Pull request merged**